pypeit/ararclines.py

```diff
@@ -54,21 +54,26 @@ def load_arcline_list(lines, disperser, spectrograph, wvmnx=None, modify_parse_d
     root = resource_filename('pypeit', 'data/arc_lines')
     with open(root+'/rejected_lines.yaml', 'r') as infile:
         rej_dict = yaml.load(infile)
-    # Loop through the NIST Tables
-    tbls = []
-    for iline in lines:
-        # Load
-        tbl = parse_nist(iline)
-        # Parse
-        if iline in parse_dict.keys():
-            tbl = parse_nist_tbl(tbl,parse_dict[iline])
-        # Reject
-        if iline in rej_dict.keys():
-            msgs.info("Rejecting select {:s} lines".format(iline))
-            tbl = reject_lines(tbl,rej_dict[iline], disperser, spectrograph)
-        tbls.append(tbl[['Ion','wave','RelInt']])
-    # Stack
-    alist = vstack(tbls)
+
+    # TODO JFH This is not elegant, but I cannot figure out how the codes reads in linelists from other directories
+    if spectrograph in ['keck_nires','keck_nirspec']:
```

Comment:
Spectrograph specific code can't enter here.
Only in the spectrograph classes.

Reply:
Yes, well there is a bug in arclines now. It does not find lamp files that are not in the 
pypeit/data/arc_lines/NIST. We don't know how to get it to look in the right place for the libelist, which is the lists. We fiddled around with the code for nearly two hours and could not figure out the logic about how these line lists are read in. It is less than transparent and undocumented.

Either @profxj or @rcooke-ast needs to help us fix this, otherwise we will need to live with this inelegant solution, or reorganize how arc_lines stores its data.

Reply:
This has been replaced with another less than elegant solution that does not reference a spectrograph, i.e. now we use the identify of the linelist. That said I think that something better could be written here.